psa_car_controller/psa/RemoteClient.py

```diff
@@ -212,13 +212,13 @@ def get_remote_access_token(self, password):
             time.sleep(60)
         return None
 
-    def horn(self, vin, count):
```

Comment:
please don't modify this file.

Reply:
I don't see harm in modifying this file? Initially I tried to not modify this file but the request would fail because of incorrect query.

Reply:
look at my suggestion https://github.com/flobz/psa_car_controller/pull/595/files#r1377882766